collect_app/src/androidTest/java/org/odk/collect/android/support/FormActivityTestRule.java

```diff
@@ -45,6 +58,12 @@ public FormEntryPage startInFormEntry() {
         return formEntryPage;
     }
 
+    private void initialCopyForm() {
+        ActivityScenario.launch(SplashScreenActivity.class);
+        FirstLaunchPage firstLaunchPage = new FirstLaunchPage().assertOnPage();
+        firstLaunchPage.clickTryCollect().copyForm(formFilename, mediaFilePaths, true);
```

Comment:
The idea of this rule is to launch `FormEntryActivity` directly without starting from the staring screen. You should just copy files here using `AdbFormLoadingUtils`

Reply:
Done.